main.py

```diff
@@ -1,5 +1,5 @@
 
 x = 1
-y = x * 2
+y = x * 2 * 3
```

Comment:
```suggestion
y = x * 2 * 3 * 5
```

Reply:
hey! thanks for the pull request. I think we should also multiply by 5

Reply:
How about just replacing 2*3*5 by 30? What do you think?

Reply:
that sounds great. go for it!